internal/cmd/keepassxctemplatefuncs.go

```diff
@@ -72,7 +72,7 @@ func (c *Config) keepassxcAttachmentTemplateFunc(entry, name string) string {
 		if err != nil {
 			panic(err)
 		}
-		tempFilename := tempDir.JoinString(name).String()
+		tempFilename := tempDir.JoinString("attachment").String()
```

Comment:
I don't think we are using goroutines now to run various resolutions in parallel, but if we ever do, this could become problematic. It might be better to somehow randomize `attachment` during execution, or perhaps make it a timestamp (`attachment-20060102030405`)?

Reply:
Good idea. Done.